decidim-admin/app/packs/stylesheets/decidim/admin/modules/_forms.scss

```
@@ -156,3 +156,13 @@ label,
     border-bottom: 1px solid grey;
   }
 }
+
+.form-single-proposal select{
```

```suggestion
.form-single-proposal select{
```
Is this class really necessary? can we get by just by using standard admin divs and classes?

In any case, the word "proposal" shouldn't appear here as this is a generic style.